test/math/FloatComplexTest.ooc

```diff
@@ -0,0 +1,67 @@
+//
+// Copyright (c) 2011-2015 Simon Mika <simon@mika.se>
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+use ooc-unit
+use ooc-math
+import math
+import FloatComplex
+
+FloatComplexTest: class extends Fixture {
+
+	complexNumber0 := FloatComplex new (2,1)
+	complexNumber1 := FloatComplex new (3,2)
+	complexNumber2 := FloatComplex new (5,3)
+
+	init: func() {
+		super("FloatComplex")
+		this add("equality", func() {
+			expect(this complexNumber0 real == 2, is true)
+			expect(this complexNumber0 imaginary == 1, is true)
```

Comment:
How about a test for `==` and `!=`, in addition?

See e.g. `FloatPoint2D.ooc` for an example of how to add equality operators.